api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirObservationDaoImpl.java

```diff
@@ -37,22 +37,14 @@
 @Component
 public class FhirObservationDaoImpl extends BaseDaoImpl implements FhirObservationDao {
```

Comment:
Also, this will cause you problems. Instead change to;
```java
@Component
public class FhirObservationDaoImpl extends BaseFhirDaoImpl<Obs> implements FhirObservationDao
```